### PR TITLE
Add initial citizenship context.

### PR DESCRIPTION
**Pull request by gannan08**
This was built on top of [work](https://project.factom.com/contexts/citizenship-v1.jsonld) done by Factom.

Thank you to Carl DiClementi (@Cahl-Dee).

### CODE REVIEW

File: contexts/citizenship-v1.jsonld [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "ctzn": "https://w3id.org/citizenship#",
+    "schema": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "Person": "ctzn:Person",
```
Comment (msporny) [inline contexts/citizenship-v1.jsonld]:
```suggestion
    "Person": "schema:Person",
```

Comment (msporny) [reply contexts/citizenship-v1.jsonld]: We are missing the following class.:

  "PermanentResidentCard": "ctzn:PermanentResidentCard"

File: contexts/citizenship-v1.jsonld [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    "birthCountry": "schema:birthCountry",
```
Comment (msporny) [inline contexts/citizenship-v1.jsonld]: All of this should probably be type scoped like we do in the base VC vocab: https://github.com/w3c/vc-data-model/blob/gh-pages/contexts/credentials/v1#L9

On the fence about that one -- feels like we'll need to type scope based on every type of card, @dlongley thoughts?

**Review by msporny — CHANGES_REQUESTED** [review]
Almost there...

File: contexts/citizenship-v1.jsonld (hunk 2 above)
Comment (dlongley) [reply contexts/citizenship-v1.jsonld]: We should type scope everything to the specific credential/card type.

File: contexts/citizenship-v1.jsonld (hunk 1 above)
Comment (gannan08) [reply contexts/citizenship-v1.jsonld]: @msporny `PermanentResidentCard` is not defined in the vocab. I can go ahead and add it to the context though.

[Person](https://digitalbazaar.github.io/citizenship-vocab/#person) does NOT refer to the schema.org Person in the spec. Is the spec wrong for this term?

File: contexts/citizenship-v1.jsonld (hunk 2 above)
Comment (dlongley) [reply contexts/citizenship-v1.jsonld]: We should also be using `@protected`.

Comment (msporny) [reply contexts/citizenship-v1.jsonld]: Let's do this now so we don't have to do it later.

File: contexts/citizenship-v1.jsonld (hunk 1 above)
Comment (msporny) [reply contexts/citizenship-v1.jsonld]: Yes, spec is wrong.

File: contexts/citizenship-v1.jsonld [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    "PermanentResidentCard": "ctzn:PermanentResidentCard",
+    "Person": "ctzn:Person",
+
+    "birthCountry": "schema:birthCountry",
+    "birthDate": {"@id": "schema:birthDate", "@type": "xsd:dateTime"},
+    "familyName": "schema:familyName",
+    "gender": "schema:gender",
+    "givenName": "schema:givenName",
+    "image": "schema:image",
```
Comment (davidlehn) [inline contexts/citizenship-v1.jsonld]:
```suggestion
    "image": {"@id": "schema:image", "@type": "@id"},
```

Comment (mattcollier) [reply contexts/citizenship-v1.jsonld]: @davidlehn we're using this field for base64 encoded images, is this compatible?

Comment (davidlehn) [reply contexts/citizenship-v1.jsonld]: Yes, assuming you are using `data:` URIs.

File: contexts/citizenship-v1.jsonld [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+
+    "id": "@id",
+    "type": "@type",
+
+    PermanentResidentCard: {
```
Comment (davidlehn) [inline contexts/citizenship-v1.jsonld]:
```suggestion
    "PermanentResidentCard": {
```

File: contexts/citizenship-v1.jsonld [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "ctzn": "https://w3id.org/citizenship#",
+        "schema": "http://schema.org/",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+        "PermanentResidentCard": "ctzn:PermanentResidentCard",
```
Comment (davidlehn) [inline contexts/citizenship-v1.jsonld]: Looks like this isn't needed within its own definition.

File: contexts/citizenship-v1.jsonld (hunk 1 above)
Comment (mattcollier) [reply contexts/citizenship-v1.jsonld]: Can someone provide clarification on this point?

File: contexts/citizenship-v1.jsonld [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+
+    "id": "@id",
```
Comment (mattcollier) [inline contexts/citizenship-v1.jsonld]: remove L 6 and 7

File: contexts/citizenship-v1.jsonld (hunk 1 above)
Comment (mattcollier) [reply contexts/citizenship-v1.jsonld]: schema:person is correct per discussion.  Resolved.

**Review by msporny — APPROVED** [review]

**Pull request merged**